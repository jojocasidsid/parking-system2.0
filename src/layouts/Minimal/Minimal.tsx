import React from 'react'
import { Outlet } from 'react-router-dom'

const Minimal = () => (
	<div className='minimal-root' style={{ height: '100vh' }}>
		<Outlet />
	</div>
)

export default Minimal
