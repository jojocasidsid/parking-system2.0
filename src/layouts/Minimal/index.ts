import Minimal from './Minimal'

export default Minimal
