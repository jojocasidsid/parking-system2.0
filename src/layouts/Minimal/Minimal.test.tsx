import React from 'react'
import renderer from 'react-test-renderer'

import {
	BrowserRouter as Router,
	Routes as Switch,
	Route,
} from 'react-router-dom'

import Minimal from './Minimal'

describe('Minimal layout', () => {
	it('should render Minimal without crashing', () => {
		const tree = renderer.create(
			<Router>
				<Switch>
					<Route path='*' element={<Minimal />} />
				</Switch>
			</Router>
		)

		expect(tree).toMatchSnapshot()
	})
})
