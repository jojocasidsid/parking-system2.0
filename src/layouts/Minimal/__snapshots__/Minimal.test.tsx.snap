// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Minimal layout should render Minimal without crashing 1`] = `
<div
  className="minimal-root"
  style={
    Object {
      "height": "100vh",
    }
  }
/>
`;
